libflux/flux-core/src/formatter/tests.rs

```diff
@@ -1231,7 +1231,7 @@ fn preserve_multiline_test() {
     return tables
         |> aggregateWindow(every: every, fn: fn, column: _column, createEmpty: true)
         |> fill(column: _column, value: defaultValue)
-        |> rename(fn: (column) => if column == _column then _as else column)"#,
+        |> rename(fn: (column) => if column == _column then _as else column)}"#,
```

Comment:
Was this a purposeful change? How did this ever work?

Reply:
yeah, this was caught after my fix :)  Earlier there was no ast check at all, code was parsing the script without checking for error and calling the formatter on the parsed script. All fixed now.